pallets/automation-price/src/weights.rs

```diff
@@ -60,14 +60,14 @@ impl<T: frame_system::Config> WeightInfo for AutomationWeight<T> {
 	fn reset_asset(v: u32, ) -> Weight {
 		(200_000_000 as Weight)
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
-			.saturating_add((9_000_000 as Weight).saturating_mul(v as Weight))
-			.saturating_add(T::DbWeight::get().reads((3 as Weight).saturating_mul(v as Weight)))
-			.saturating_add(T::DbWeight::get().writes((6 as Weight).saturating_mul(v as Weight)))
+			.saturating_add((20_000_000 as Weight).saturating_mul(v as Weight))
+			.saturating_add(T::DbWeight::get().reads((4 as Weight).saturating_mul(v as Weight)))
+			.saturating_add(T::DbWeight::get().writes((5 as Weight).saturating_mul(v as Weight)))
 	}
 	fn update_asset_reset() -> Weight{
 		(200_000_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(1 as Weight))
-			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
```

Comment:
got the number of reads and writes flipped and update_asset_reset is also used in reset_asset function